cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -73,6 +73,8 @@ type Evaluate struct {
 	Configuration string `long:"configuration" description:"Configuration file to set up an evaluation run."`
 	// ExecutionTimeout holds the timeout for an execution.
 	ExecutionTimeout uint `long:"execution-timeout" description:"Execution timeout for compilation and tests in minutes." default:"5"`
+	// RunIDOffset holds the offset increment for the run id used in creating the result folders.
+	RunIDOffset uint `long:"run-id-starts-at" description:"Sets the offset for the ID used in creating result folders." default:"1"`
```

Comment:
Should we set the default to "0" so it does not affect anything unless you set it?

Reply:
Yeah the default should be zero. The offset is basically the number of runs that are already run by some other job.